### ISSUE TITLE
Falldatenblatt: Strafrecht

### ISSUE THREAD

**Issue by PBaumfalk** [jlawyerorg/j-lawyer-forms#6]
![image](https://user-images.githubusercontent.com/50404378/96607594-cae31080-12f8-11eb-98d2-b2bede0dfac8.png)

Wir haben wieder ein Falldatenblatt entwickelt.

**Comment by iradraconis** [comment]
Da ich mit Falldatenblättern bisher noch gar nicht gearbeitet habe, würde mich interessieren wie diese in den Arbeitsablauf integriert werden können. Gibt es irgendwo Hinweise dazu? So wie ich es verstanden habe, liegt der Zweck in der Erfassung sachgebietsbezogener Daten zur Akte. Aber wie liest man diese aus oder verwendet sie weiter? Über Platzhalter in Vorlagen?

**Comment by j-dimension** [comment]
@iradraconis ja, man kann alle Informationen in den Falldatenblättern auch in Dokumenten verwenden (einfach mal das "?" in einem Falldatenblatt klicken, da werden alle Platzhalter angezeigt). Außerdem können die Falldatenblätter beliebig "intelligent" sein - bspw. Berechnungen durchführen etc. 

Hier ist ein (relativ technischer) Einstieg:
https://www.j-lawyer.org/?p=3601

**Comment by PBaumfalk** [comment]
Platzhalter und entsprechende Vorlagen.

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

Sent by Mobile

**Comment by iradraconis** [comment]
Also speziell das Falldatenblatt Strafrecht dient also eher dazu, eine Art "Vermerk" bei Mandatsannahme zu erstellen? Das heißt ich hätte im besten Fall bei Verwendung des Falldatenblatts schon eine Akte angelegt? 

Konkret in diesem Falldatenblatt würden mir noch zahlreiche Daten fehlen. 
- Welche Behörde (Pol, StA), welches Gericht, 
- Aktenzeichen (je Beteiligte)
- Angehörige, die informiert werden müssen/dürfen und wo eine Entbindung der Schweigepflicht geklärt sein muss, 
- die Frage nach der Anklage ist überflüssig, wenn man diese ersetzt durch "Verfahrensstand" (EV, ZV, HV, I. Instanz, II. Instanz, III. Instanz, Vollstreckungsverfahren), vielleicht als Auswahldialog wenn abschließend?
- die Frage nach der Akteneinsicht ist im Strafrecht auch eher überflüssig, da ohne Akteneinsicht ohnehin keine Verteidigung möglich ist, besser vielleicht "ggfs. Auftrag"?
- Anpassung des Falldatenblatts auf die Variante Opfervertretung/Nebenklage

So meine ersten Gedanken dazu.. Ich glaube ich beginne zu erkennen, wie nützlich diese Falldatenblätter sein können.

**Comment by PBaumfalk** [comment]
Behörde ist in den Beteiligten und können über die Gruppen eingefügt werden. Die entsprechenden Platzhalter gibt es schon. Über die Gruppen kann man StA, Polizei und Gericht direkt in die Akte eintragen.

Aktenzeichen gibt es auch bei den Beteiligten.

Akteneinsicht ist eher eine Hilfe für den schnellen Blick (wurde die Akte schon übersandt?).

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

Sent by Mobile

**Comment by stm9x9** [comment]
> Behörde ist in den Beteiligten und können über die Gruppen eingefügt werden. Die entsprechenden Platzhalter gibt es schon. Über die Gruppen kann man StA, Polizei und Gericht direkt in die Akte eintragen.

Was meinst Du mit Gruppen hier?

**Comment by stm9x9** [comment]
> Wir haben wieder ein Falldatenblatt entwickelt.

Wird das automatisch in die Liste der installierbaren Datenblätter für den Admin eingepflegt? Weil ich finde das Falldatenblatt nicht.
